lib/mix/tasks/format.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Mix.Tasks.Format do
+  use Mix.Task
+
+  alias ExFormat
+
+  @shortdoc "Formats Elixir source code"
+
+  def run(file_paths) do
+    Enum.map(file_paths, fn file_path ->
+      formatted =
+        file_path
+        |> File.read!()
+        |> ExFormat.format()
+      file = File.open!(file_path, [:write])
+      IO.binwrite(file, formatted)
+      File.close(file)
+      IO.puts("Formatted #{file_path}")
+    end)
```

Comment:
A few notes:

* you can use `File.write!/2` instead of open/write/close
* `Enum.each/2` is better than `Enum.map` here because it will not build a list that is never used
* use `Mix.shell().info("Formatted #{file_path}")` instead of `IO.puts/1`. Using the Mix shell ensures that this will work when the shell is changed at runtime and a bunch of other stuff.